client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-description.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { translate } from 'i18n-calypso';
+
+class DomainConnectAuthorizeDescription extends Component {
+	placeholder = () => {
+		return (
+			<div className="domain-connect__is-placeholder">
+				<span></span>
+				<span></span>
+				<span></span>
+			</div>
+		);
+	}
+
+	getDescription = () => {
+		const { providerId } = this.props;
+
+		switch ( providerId ) {
+			case 'g-suite':
+				return translate( 'Howdy! It looks like you want to make your domain work with the ' +
```

Comment:
Hmm, seems like all those descriptions are the same - just the service's name changes. I'd hate it if some angry translator killed you before meeting your in person at the GM, so maybe you could make those service names placeholders? 😉 It would simplify this method too.